api/api-rules/violation_exceptions.list

```diff
@@ -393,6 +393,9 @@ API rule violation: names_match,k8s.io/api/core/v1,RBDVolumeSource,RBDPool
 API rule violation: names_match,k8s.io/api/core/v1,RBDVolumeSource,RadosUser
 API rule violation: names_match,k8s.io/api/core/v1,VolumeSource,CephFS
 API rule violation: names_match,k8s.io/api/core/v1,VolumeSource,StorageOS
+API rule violation: names_match,k8s.io/api/networking/v1alpha1,ParentReference,Resource
+API rule violation: names_match,k8s.io/api/networking/v1alpha1,ServiceCIDRConfigSpec,IPv4CIDR
+API rule violation: names_match,k8s.io/api/networking/v1alpha1,ServiceCIDRConfigSpec,IPv6CIDR
```

Comment:
we shouldn't add new violations, right?

Reply:
I'm just trying to make it work :smile: , there are a lot of things here that are magic to me,